container.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dashboard</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+    
+    <link rel="stylesheet" href="container.css">
+</head>
+<body>
+  <div class="container"> 
+    <!-----Navbar-------> 
+    <nav class="navbar">
+        <div class="left-navbar-items">
+            <a href="#" class="navbar-brand">
+               <img class="logo" src="assets/images/hr-logo.png" alt="Logo" >
+               <span id="magazine">Magazine</span>
+            </a>
+            <p id="user-type">Admin</p>
+        </div>
+        <div class="right-navbar-icons">
+            <button type="button" class="icon-btn" title="Toggle Theme" id="theme-toggle-btn">
+                <i class="fas fa-sun"></i> 
+            </button>
+            <img class="avatar" src="assets/images/avatar-default.png" alt="Avatar" >
+                <!-- Speech Bubble Content -->
+                <div class="speech-bubble hidden">
+                     <img src="assets/images/avatar-default.png" alt="Avatar" class="speech-avatar">
+                     <div class="speech-name">Username</div>
+                          <div class="speech-options">
+                              <div class="profile-icon"><i class="fas fa-user"></i><span>Profile</span></div>
+                              <button type="button">Edit Profile</button>
+                          </div>
+                          <hr>
+                          <button type="button" class="sign-out-btn"><i class="fas fa-sign-out-alt"></i>Sign Out</button>
+                          <p class="speech-footer">Privacy Policy . Terms . Cookies</p>
+                </div> 
+        </div>
+    </nav>
+    <!----End Navbar---->
+    <div class="content-aside-wrapper">
+        <!------------------------------------------------------------------------------------------------------>
+        <!-- Sidebar Start -->
+        <aside class="sidebar">

```

Comment:
It is my opinion, but you may think of it, I don't like the vertical line that appears in the side bar. personally, I will add icons that will look much better